t/leak.t

```diff
@@ -260,7 +260,7 @@ sub alert {
 sub leaktest {
     my $test = shift;
     my %arg  = (init=>10, test=>100, max_mem_diff=>288, diag=>0, @_);
-    my $tmp = 'x' x 1000000; undef $tmp;
+    my $tmp = ('x' x 1000000).localhost; undef $tmp;
```

Comment:
I suppose this is a typo and you meant `localtime`?

Reply:
And why not just `time`?
Also, a comment about reason would make this change more clear.

Reply:
I'll update this PR later today.